specs/protocol/superchain-configuration.md

```diff
@@ -64,3 +67,54 @@ When the Pause is activated, the following methods are disabled:
 1. `StandardBridge.finalizeBridgeERC20()`
 1. `StandardBridge.finalizeBridgeETH()`
 1. `L1ERC721Bridge.finalizeBridgeERC721()`
+
```

Comment:
i think we should also add something on the Overview section about this:

"The SuperchainConfig contract is used to manage global configuration values for multiple OP Chains within
a single Superchain network and also keep track of the dependency set of the graph."

Or something like that